python/ucxx/_lib_async/endpoint.py

```diff
@@ -445,12 +445,24 @@ def get_ucp_worker(self):
         """
         return self._ctx.worker.handle
 
+    def get_ucxx_worker(self):
+        """Returns the underlying UCXX worker pointer (ucxx::Worker*)
+        as a Python integer.
+        """
+        return self._ctx.worker.ucxx_ptr
+
     def get_ucp_endpoint(self):
         """Returns the underlying UCP endpoint handle (ucp_ep_h)
         as a Python integer.
         """
         return self._ep.handle
 
+    def get_ucxx_endpoint(self):
+        """Returns the underlying UCXX endpoint pointer (ucxx::Endpoint*)
+        as a Python integer.
+        """
+        return self._ep.ucxx_ptr
```

Comment:
**question:** as above, is there a particular reason that the async-wrappers provide all of these accessors as functions, rather than properties?

Reply:
No, see my reply https://github.com/rapidsai/ucxx/pull/122/files#r1395674667 .